src/SourceBuild/patches/msbuild/0001-Flow-live-version-of-System.Security.Permissions-for.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Matt Thalman <mthalman@microsoft.com>
+Date: Wed, 23 Aug 2023 16:20:02 -0500
+Subject: [PATCH] Flow live version of System.Security.Permissions for
+ source-build
+
+Backport: https://github.com/dotnet/msbuild/pull/9158
+---
+ eng/Version.Details.xml | 5 +++++
+ 1 file changed, 5 insertions(+)
+
+diff --git a/eng/Version.Details.xml b/eng/Version.Details.xml
+index 244fed20fb..91137a797c 100644
+--- a/eng/Version.Details.xml
++++ b/eng/Version.Details.xml
+@@ -37,6 +37,11 @@
+       <Uri>https://github.com/dotnet/runtime</Uri>
+       <Sha>d099f075e45d2aa6007a22b71b45a08758559f80</Sha>
+     </Dependency>
++    <!-- Necessary for source-build. This allows the live version of the package to be used by source-build. -->
++    <Dependency Name="System.Security.Permissions" Version="7.0.0">
++      <Uri>https://github.com/dotnet/runtime</Uri>
++      <Sha>d099f075e45d2aa6007a22b71b45a08758559f80</Sha>
++    </Dependency>
```

Comment:
@mthalman for my understanding of what this does:

When building the msbuild repo, when the build normally uses the `System.Security.Permissions` version `7.0.0` package, during source-build, it will use the _live version_ package instead.

Does _live version_ mean: the latest version available?
What are the Uri and Sha for?

Reply:
> Does live version mean: the latest version available?
> What are the Uri and Sha for?

Generally speaking it means either

1. The current version being source-built - if the dependent package was already built during the full source build
2. The n-1 version coming from the previous artifacts - if the dependent package was not already built during the full source build.  If this is the case, the reference package is not allowed to be bundled as that would break the source build rules and would get detected by the poison infrastructure.